File: Selenium/Project/ProjectActivity5.py
# Import webdriver from selenium

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Set up the Firefox Driver with WebDriverManger
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL

    driver.get("https://alchemy.hguy.co/jobs/")

    # Find the navigation bar element
    navigationBar = driver.find_element(By.ID("menu-item-24"))

    # Find the "Jobs" menu item and click it
    jobsMenuItem = navigationBar.find_element(By.LINK_TEXT("Jobs"))
    jobsMenuItem.click()

    # Read the page title
    pageTitle = driver.Title()

    # Verify the correct page using the page title
    expectedTitle = "Jobs – Alchemy Jobs"
    assert  == expectedTitle,f"page title is '{pageTitle}',but expected '{expectedTitle}'"

    # Print success message if the verification is successful
    print("You are on the correct page.")

File: Selenium/Project/ProjectActivity5.java
package ProjectActivities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.*;

public class ProjectActivity5 {
    WebDriver driver;
    @BeforeClass
    public void setUp() {
        WebDriverManager.firefoxdriver().setup();
        // Create a new instance of the Firefox driver
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");
    }
    @Test
    public void testJobNavigation(){

        // Find the navigation bar element
        WebElement navigationBar = driver.findElement(By.id("menu-item-24"));

        // Find the "Jobs" menu item and click it
        WebElement jobsMenuItem = navigationBar.findElement(By.linkText("Jobs"));
        jobsMenuItem.click();

        // Read the page title
        String pageTitle = driver.getTitle();

        // Verify the correct page using the page title
        String expectedTitle = "Jobs – Alchemy Jobs";
        Assert.assertEquals(pageTitle, expectedTitle, "Incorrect page title");

        // Print success message if the verification is successful
        System.out.println("You are on the correct page.");
    }

    @AfterClass
    public void tearDown(){
        driver.quit();
    }
}